README.md

```diff
@@ -27,24 +15,17 @@ These instructions will build and run a Docker container with a Grafana instance
 
 2. Unzip into plugins directory.
```

Comment:
We should provide the typical plugins directory location here

Reply:
That's provided in the information a few lines above.